saleor/graphql/product/mutations/products.py

```diff
@@ -461,6 +469,14 @@ class Meta:
         description = 'Updates a product image.'
         model = models.ProductImage
 
+    @classmethod
```

Comment:
You should not write the same code twice.

Reply:
True, `ProductImageUpdate` could inherit from `ProductImageCreate`.